v2/file-format-conversion/src/test/java/com/google/cloud/teleport/v2/templates/FileFormatConversionTest.java

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright (C) 2019 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.google.cloud.teleport.v2.templates;
+
+import com.google.cloud.teleport.v2.templates.FileFormatConversion.FileFormatConversionOptions;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.testing.TestPipeline;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+/** Test cases for the {@link FileFormatConversion} class. */
+public class FileFormatConversionTest {
```

Comment:
I am wondering if we can/should add unit tests going from X -> Y given that we have .avro and .parquet and .csv test resources already.

Reply:
Those files are in the common module... Also I can't use write builders in test cases as they create files outside the src folder in the module.

Reply:
Can you copy the relevant test resources to the file-format-conversion module?